.github/workflows/linkchecker.yml

```diff
@@ -7,7 +7,7 @@ name: Check markdown links on modified website files
 jobs:
   vercel-deployment-poll:
     runs-on: ubuntu-latest
-    timeout-minutes: 3 #cancel job if no deployment is found within x minutes
+    timeout-minutes: 5 #cancel job if no deployment is found within x minutes
```

Comment:
Bumped this since builds are hovering around 3 minutes, so I was getting some flaky CI passes.